packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -161,11 +161,11 @@ class Send extends Component {
   preValidate = values => {
     const { balance, token } = this.props;
     const amount = +values.amount;
-    const amountBn = new BigNumber(amount);
+    const amountBn = new BigNumber(amount.toString());
 
-    if (!amount || isNaN(amount)) {
+    if (!amount || amountBn.isNaN()) {
```

Comment:
`!amount` is not useful here, since `amount.toString()` would throw an error before.

Could you add the test `if (!amount) {...}` before the `new BigNumber` initialisation?

Reply:
i've pushed a commit with this change